scheduler/test/cook/test/kubernetes/compute_cluster.clj

```diff
@@ -6,7 +6,8 @@
             [cook.mesos.task :as task]
             [cook.scheduler.scheduler :as sched]
             [cook.test.testutil :as tu]
-            [datomic.api :as d])
+            [datomic.api :as d]
+            [clojure.tools.logging :as log])
```

Comment:
Lexicographic

Reply:
logging wasn't needed, so deleted instead

Reply:
I still see it here

Reply:
deleted now (merge conflict issue)